roles/network_plugin/calico/tasks/check.yml

```diff
@@ -67,8 +59,6 @@
       Minimum version is {{ calico_min_version_required }} supported by the previous kubespray release.
       But current version is {{ calico_version_on_server.stdout }}.
   when:
-    - 'calico_version_on_server.stdout is defined'
```

Comment:
I think it should be keep here

Reply:
Previously, `calico_version_on_server.stdout` is always undefined as the check command is wrong.

If the `check version` is required, the stdout is expected to be an exact version. Otherwise, the assertion would fail.

If we allow/ignore the stdout is undefined, it seems there is no need to check the version anymore.